.github/workflows/run_tartan.yml

```diff
@@ -57,4 +61,87 @@ jobs:
       - name: Build portDNN
         uses: ./.github/actions/build_portDNN_action
         with:
-          workspace: ${{ github.workspace }}
\ No newline at end of file
+          workspace: ${{ github.workspace }}
+
+      - name: Package Build Artifacts
+        run: |
+          tar -cvzf artifacts.tar.gz build portDNN_build_dir portBLAS_build_dir
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: ock_demo_build
+          path: artifacts.tar.gz
+
+  build_and_run_networks:
```

Comment:
not sure where to mention this, but I don't think we want to put the pictures in the ock repo (fine as part of an artifact).

Reply:
Its not straight forward to get the images from image-net. https://www.image-net.org/challenges/LSVRC/index.php
I can reduce the number of images.

Reply:
would it not run in a different way with less images. I'm just not sure I want these binary objects in OCK.